tests/conftest.py

```diff
@@ -87,11 +87,9 @@ def databricks_http_cluster_target():
         "token": os.getenv("DBT_DATABRICKS_TOKEN"),
         "method": "http",
         "port": 443,
-        # more retries + longer timout to handle unavailability while cluster is restarting
-        # return failures quickly in dev, retry all failures in CI (up to 5 min)
-        "connect_retries": 5,
-        "connect_timeout": 60,
-        "retry_all": bool(os.getenv("DBT_DATABRICKS_RETRY_ALL", False)),
+        "connect_retries": 3,
+        "connect_timeout": 5,
+        "retry_all": False,
```

Comment:
with the below conftest fixture I _think_ we don't need to have all these retries anymore

Reply:
If I understand this correctly, this will result in failure in more scenarios, not less. So if we're wrong, the tests simply won't pass. I'm fine with that since we'd be alerted.

Reply:
yea most of the time when these fail it's because something is actually broken